### DISCUSSION TITLE
Password generation

### DISCUSSION THREAD

**Discussion by artdome** [lockerpm/.github#12] [Ideas]
I would like to see the history of generated passwords, it is very convenient when you have a lot of passwords in the storage, sometimes you need to look at previously generated passwords.

**Comment by fabo123456** [comment]
Hi
You can view the history already. 
On Android for example there is "view password history" if you have changed the password at any time

**Reply by artdome** [reply]
That's nice,
but I meant if I just need to quck generate any password to put it on account I started create, and then I wish to save that generated password to my Locker account. So I would love to have such option to see 5 last generated password for example.

**Reply by luongnh261** [reply]
Hello @arthurwrls, this feature request has been added to Locker's product backlog for future improvement. Kindly wait for more updates of our 2025 Backlog soon.
